nix/dbx/dogebox.nix

```diff
@@ -64,7 +64,7 @@ in\
         export NIX_PATH=nixpkgs=/nix/var/nix/profiles/per-user/root/channels/nixos:nixos-config=/etc/nixos/configuration.nix:/nix/var/nix/profiles/per-user/root/channels
         export PATH=$PATH:${pkgs.git}/bin
         echo "Adding nixpkgs channel..."
-        ${pkgs.nix}/bin/nix-channel --add https://nixos.org/channels/nixos-24.11 nixpkgs
+        ${pkgs.nix}/bin/nix-channel --add https://nixos.org/channels/nixos-24.11-small nixpkgs
```

Comment:
I think we should make this change, but also not do it in this PR. The current workflow that most people (me) are using is a non-small 24.11 VM. It seems like the full and the `-small` channel don't more in lockstep, and so that may require even _more_ additional rebuilding at boot time if there's a mismatch between versions?